Update docs on Dependabot usage (#2954)
alphagov/govuk-developer-docs: source/manual/manage-ruby-dependencies.html.md

parent: "/manual.html"
---

We're [obliged to keep our software current][current]. To help with this, we use a
service called [Dependabot][] to perform automated dependency upgrades.

[RFC 126][] describes the custom configuration we have for Dependabot to reduce the

> number of open PRs, number of deployments and effort required to ensure apps always use
> the latest version of libraries that we care about.

Reviewer: ```suggestion
number of PRs it opens, and therefore the number of deployments and effort required to keep our apps up to date.
```

### Reviewing Dependabot PRs

Dependabot updates occur relatively soon after a new version is published, which means
there’s a risk of updating to a rogue version. Some updates also contain breaking

> changes, irrespective of when they are published; using Dependabot over “bundle update”
> makes it easier to spot these.

Reviewer: I don't think the rest of this sentence is very relevant to the rest of the section - we're just setting the context on why reviewing Dependabot PRs properly is important.

```suggestion
changes, irrespective of when they are published.
```
